src/themes/teams-dark/components/Text/textVariables.ts

```diff
@@ -0,0 +1,23 @@
+export interface ITextVariables {
+  importantTextColor: string
+  atMentionMeTextColor: string
+  atMentionOtherTextColor: string
+  disabledTextColor: string
+  errorTextColor: string
+  successTextColor: string
+  timestampTextColor: string
+  timestampHoverTextColor: string
+}
+
+export default (siteVariables): ITextVariables => {
+  return {
+    atMentionOtherTextColor: siteVariables.brand06,
+    atMentionMeTextColor: siteVariables.orange04,
+    disabledTextColor: siteVariables.gray06,
+    errorTextColor: siteVariables.red,
+    successTextColor: siteVariables.green04,
+    timestampTextColor: siteVariables.gray04,
+    timestampHoverTextColor: siteVariables.gray02,
+    importantTextColor: siteVariables.red,
```

Comment:
can these either be sorted alphabetically or the same as each other?

Reply:
Let's also remove the duplicate `Text` word in each.  Variables are already namespaced by display name:

```jsx
theme.componentVariables.Text().successColor
```